src/types/native.ts

```diff
@@ -30,6 +30,7 @@ export default interface NativeRNCloudStorage {
   createDirectory: (path: string, scope: NativeRNCloudCloudStorageScope) => Promise<void>;
   listFiles: (path: string, scope: NativeRNCloudCloudStorageScope) => Promise<string[]>;
   readFile: (path: string, scope: NativeRNCloudCloudStorageScope) => Promise<string>;
+  triggerDownloadFile: (path: string, scope: NativeRNCloudCloudStorageScope) => Promise<boolean>;
```

Comment:
Generally, `downloadFile` instead of `triggerDownloadFile` should be sufficient to clarify what the function does (combined with the documentation), no?